eosc/cmd/root.go

```diff
@@ -47,7 +47,7 @@ func init() {
 	RootCmd.PersistentFlags().BoolP("debug", "", false, "Enables verbose API debug messages")
 	RootCmd.PersistentFlags().StringP("vault-file", "", "./eosc-vault.json", "Wallet file that contains encrypted key material")
 	RootCmd.PersistentFlags().StringSliceP("wallet-url", "", []string{}, "Base URL to wallet endpoint. You can pass this multiple times to use the multi-signer (will use each wallet to sign multi-sig transactions).")
-	RootCmd.PersistentFlags().StringP("api-url", "u", "https://mainnet.eoscanada.com", "API endpoint of eos.io blockchain node")
+	RootCmd.PersistentFlags().StringP("api-url", "u", "http://localhost:8888", "API endpoint of eos.io blockchain node")
```

Comment:
Frankly, I'm far from seeing this as a good move. This makes the out of the box experience much to be desire. What about asking EOS Nation of Greymass if they would agree that their API is the default value?

I would even prefer no value at all and a good error message that explains how to properly setup `eosc`. 

Here what would look like a default `get_info` invocation:

```
eosc get info
ERROR: get info: http://localhost:8888/v1/chain/get_info: Post http://localhost:8888/v1/chain/get_info: dial tcp [::1]:8888: connect: connection refused
```

It's not really helpful and super hard to understand what needs to be modified exactly. I get the idea of using `localhost:8888`, but I don't like it.

Reply:
eosc is not only for EOS mainnet, therefore having another API provider does not seem like a good solution either. I can add a good error message!

Reply:
Of course it's usable with all other chains, but still, I bet most usages are made against EOS Mainnet.

Wallet usually list EOS Mainnet as their primary option. They also do not have the exact same problem since there is a bootstrap phase that ask to configure the wallet correctly.

We should probably aim to that kind of workflow also at some point.

Anyway, I don't think I will change the decision, it's too late I guess. 

A better error message is the minimum, hopefully it does not trickle down too much everywhere.